core/include/webview/detail/backends/win32_edge.hh

```diff
@@ -868,12 +878,18 @@ private:
         DispatchMessageW(&msg);
       }
     }
+    // add the default window size event back to the event queue
+    if (!m_is_window_shown) {
+      set_is_size_set(false);
+      dispatch_size_default(m_owns_window);
+    }
   }
 
   // The app is expected to call CoInitializeEx before
   // CreateCoreWebView2EnvironmentWithOptions.
   // Source: https://docs.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/webview2-idl#createcorewebview2environmentwithoptions
   com_init_wrapper m_com_init;
+  void *window_ptr = nullptr;
```

Comment:
I understand this variable is only used during construction so I would prefer a shorter lifespan. We already have `m_window`.

Maybe it could at least be renamed to something like `m_hwnd_or_phwnd` to make it more clear.

The passed-in window can be either a native window handle (HWND, which is a pointer), or, for backward compatibility reasons, `HWND*`.

Reply:
I take your point.
My eye was on standardised methods across all platforms, so I have simplified it across all by passing `*window` to `window_init(window)` and handling it further for each platform therein.

See latest commit.